client/app/scripts/liveblog-syndication/reducers/incoming-syndication.js

```diff
@@ -15,12 +15,18 @@ liveblogSyndication
                 case 'ON_SAVED_POST':
                 case 'ON_REMOVED_POST':
                     return angular.extend(state, {
+                        notification: action.notification || null,
                         posts: angular.extend(state.posts, {
                             _items: state.posts._items.filter(function(item) {
                                 return (item._id != action.post._id);
                             })
                         })
                     });
+
+                case 'ON_FLUSHED_NOTIF':
```

Comment:
Seems a bit weird or unnecessarily indirect to handle the flushing of a notification with a notification. Is this necessary? Please apply KISS principles.